object-store/1.2/modules/ROOT/pages/object-store-to-define-a-new-os.adoc

```diff
@@ -1,46 +1,61 @@
 = Define Object Stores Examples - Mule 4
```

Comment:
This is from the Term Presentation Reference - I don't know if it applies to this doc:

object store (nonbranded: an object store, the Mule object store, the object store included in Mule...)
Object Store (branded internal service that does not require version)